### PR TITLE
Fix: Fixed Validation of Center name

### PR DESCRIPTION
**Pull request by therajanmaurya**
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `MifosStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.
…er if any validation fails

### CODE REVIEW

**Comment by ishan1604** [comment]
@therajanmaurya when writing commit message. 

You don't need to specify **Fixed** again in it. So you could rephrase this as 
fix: Validation of center name .....

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/online/createnewcenter/CreateNewCenterFragment.java [hunk 1]
```diff
@@ -177,6 +178,7 @@ public void onDatePicked(String date) {
     }
 
     public boolean isValidCenterName() {
```
Comment (ishan1604) [inline mifosng-android/src/main/java/com/mifos/mifosxdroid/online/createnewcenter/CreateNewCenterFragment.java]: method name should be 

```
isCenterNameValid()
```

**Comment by therajanmaurya** [comment]
@ishan1604 I have made new PR that fixes all the things.
